parking/src/models/Slot.java

```diff
@@ -0,0 +1,149 @@
+package models;
+
+import java.util.*;
+import models.ParkingLot;
+import models.Vehicle;
+import constant.VehicleType;
+import models.Time;
+
+public class Slot {
+	public static Map<String, Integer> FloorCorrespondingToslotId = new HashMap<String, Integer>();
+	public static Map<String, Integer> SlotCorrespondingToslotId = new HashMap<String, Integer>();
+	public static Map<String, String> SlotIdCorrespondingToRegNumber = new HashMap<String, String>();
+	public static Map<String, Time> TimeCorrespondingToSlotId = new HashMap<String, Time>();
+	
+	public static String slotId(Vehicle vehicle) {	
+   	
+		VehicleType.VehicleTypes vType =  vehicle.getVehicleType();	
+   	//Id generator
+	String slotId = UUID.randomUUID().toString();
+	    
+ 	//try to give slot
+   	 boolean checkSlotOccupied = ParkingLot.occupied.contains(slotId); 
+        if (checkSlotOccupied)
+        {
+       	 System.out.println("slotoccupied");
+        }
+        else
+        {   	 if(vType == VehicleType.VehicleTypes.twoWheeler)

```

Comment:
if else conditions should not be used here, use proper data structures and pull data directly from them.